src/signals/incident-management/components/HiddenInput/index.tsx

```diff
@@ -1,12 +1,21 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2019 - 2022 Gemeente Amsterdam
+
+/**
+ * Deprecated note:
+ * This component should be removed when DefaultTextAdmin is removed and all municipalities are using StandardTextAdmin.
+ */
+
 import type { FC } from 'react'
 
 interface HiddenInputProps {
   name: string
   value: string
 }
 
+/**
+ * @deprecated
```

Comment:
why is this added? should it be removed?

Reply:
It's added to prevent someone using it. If someone tries to use it they will get a "this component is deprecated" warning.